pyproject.toml

```diff
@@ -178,6 +178,7 @@ markers = [
     "disable_qthread_pool_start: Disable strarting QRunnable using QThreadPool start in this Test",
     "disable_qtimer_start: Disable timer start in this Test",
     "disable_qanimation_start: Disable animation start in this Test",
+    "slow: Time consuming test",
```

Comment:
because I added tests that cover the datasets which are very slow, I decided to add this marker so that we can avoid testing these regularly.